### DISCUSSION TITLE
Can't log in despite correct name and password

### DISCUSSION THREAD

**Discussion by Flugzahn** [ppy/osu#22007] [Q&A]
Hey,
Last week I was logged in automatically (osu! lazer) but now when I try to log in it loads briefly and then nothing happens. The password and username are correct, because in osu! I can log in normally.
Does anyone know where the problem is? 

Edit: I disabled ipv6 but i still got this problem.
[performance.log](https://github.com/ppy/osu/files/10333096/performance.log)
[runtime.log](https://github.com/ppy/osu/files/10333097/runtime.log)
[database.log](https://github.com/ppy/osu/files/10333098/database.log)
[network.log](https://github.com/ppy/osu/files/10333099/network.log)

**Comment by bdach** [comment]
Please post logs as per https://github.com/ppy/osu/discussions/15188

**Reply by Flugzahn** [reply]
Done ✔️

**Reply by bdach** [reply]
Log shows this which is weird:

```
2023-01-02 18:26:09 [verbose]: Response was: {"authentication":"basic"}
2023-01-02 18:26:09 [verbose]: Failing request osu.Game.Online.API.Requests.GetUserRequest (System.Net.WebException: Unauthorized)
```

- Can you access the osu! website?
- What's your profile username?

**Reply by Flugzahn** [reply]
Yupp, i can access to the Osu Website and im logged in.

Username: Flugzahn

**Reply by bdach** [reply]
I can't see a profile with that username (shows "user not found"). Are you currently under a restriction? This may very well be related to this error.

**Reply by Flugzahn** [reply]
Yupp im still restricted cause Osu is not responding...

But i installed osu lazer 2 weeks ago and i could login successfully and in osu! ofc too.

(im since October restricted)

**Reply by bdach** [reply]
I believe there may have been some recent changes to osu-web with respect to how restrictions are handled in API, but the web team or @peppy would probably know more.

**Reply by Flugzahn** [reply]
Alright,

I'll keep you updated if by restriction is over.

**Reply by peppy** [reply]
@nanaya could anything have changed to break restricted user login on the oauth side? (lazer.ppy.sh)

**Reply by nanaya** [reply]
that's because of the pr which completely disables oauth for restricted user was picked and deployed for lazer branch. The pr has since been reverted but not on lazer. I've updated the branch which should fix it.

**Reply by peppy** [reply]
This should now be deployed FWIW.
